src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Represents a task to be completed.
+ */
+public class Task {
+    protected String details;
+    protected boolean isCompleted;
+
+    /**
+     * Constructor for the task.
+     * @param details task details.
+     */
+    public Task(String details) {
+        this.details = details;
+    }
+
+    /**
+     * To indicate that the task is completed (with a tick symbol).
+     */
+    public void markAsCompleted() {
```

Comment:
Perhaps you can simplify from markAsCompleted to complete or completeTask?

Reply:
Yep thanks!